packages/core/src/Stores/client-store.js

```diff
@@ -81,6 +81,7 @@ export default class ClientStore extends BaseStore {
     device_data = {};
     is_authorize = false;
     is_logging_in = false;
+    is_client_initialized = false;
```

Comment:
nit: This name is to general.
I personally dont understand what kind of initialization it means!?
I suggested to be more specific in defining the name.

Reply:
@ali-hosseini-deriv , sure. I mean here, that client store is loaded.
I'll change the name accordingly